src/main/resources/patchConfig.xml

```diff
@@ -142,10 +142,13 @@
     <module name="FileLength">
         <property name="fileExtensions" value="java"/>
     </module>
+    <!--<module name="LineLength">-->
+        <!--<property name="fileExtensions" value="java"/>-->
+        <!--<property name="max" value="100"/>-->
+        <!--<property name="ignorePattern" value="^ *\* *([^ ]+|\{@code .*|&lt;a href=&quot;[^&quot;]+&quot;&gt;)$"/>-->
+    <!--</module>-->
     <module name="LineLength">
-        <property name="fileExtensions" value="java"/>
-        <property name="max" value="100"/>
-        <property name="ignorePattern" value="^ *\* *([^ ]+|\{@code .*|&lt;a href=&quot;[^&quot;]+&quot;&gt;)$"/>
+        <property name="max" value="20"/>
```

Comment:
this is over demanding, please make it 80 or 90.
We do not need storm of violations, we need few.

Please remove keep fileExtensions and ignorePattern.

Reply:
done